types/conventional-changelog-core/conventional-changelog-core-tests.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable:no-mergeable-namespace no-namespace */
+"use strict";
+
+import conventionalChangelogCore from "conventional-changelog-core";
+
+namespace Test {
```

Comment:
Is there a reason for having a namespace here? You should be able to test without it. 

Same question for the other test files.

Reply:
I use this approach since I want to isolate the parts of the definition to make the tests easier to identify.

For example in [/types/conventional-recommended-bump/conventional-recommended-bump-tests.ts#L34](https://github.com/JasonHK/DefinitelyTyped/blob/conventional-changelog/types/conventional-recommended-bump/conventional-recommended-bump-tests.ts#L34):

```typescript
namespace Test.Options {
    ...
}
```

The namespace `Test.Options` means I was testing `conventionalRecommendedBump.Options` inside of it.

If this is acceptable should I leave a comment inside the files to prevent confusion by others?

Reply:
It's not unacceptable if it's easier for you to read. It's a little bit confusing as it's not the intended purpose of namespaces. 

I'm ok with it, I'll merge it.